src/app/new-reports/page.jsx

```diff
@@ -0,0 +1,120 @@
+"use client"
+import React, { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import Link from "next/link";
+import axios from 'axios';
+import styles from './newReportStyles';
+
+
+
+const NewReport = () => {
+     const [image, setImage] = useState(null);
+     const [title, setTitle] = useState('');
+     const [description, setDescription] = useState('');
+     const router = useRouter();
+
+
+const handleImageUpload = (event) => {
+     const file = event.target.files[0];
+     if(file) {
+          const reader = new FileReader();
+          reader.onload = () => {
+               setImage(reader.result);
+          };
+          reader.readAsDataURL(file);
+     };
+}
+
+const handleImageClick = () => {
+     document.getElementById('upload-button').click();
+   };
+
+const handleSubmit = async() => {
+     console.log({image, title, description});
+     const reportData ={
+          title,
+          image,
+          description
+     };
+
+     try{
+          const response = await axios.post('http://localhost:3001/reports', reportData);
```

Comment:
You can export this logic into service and use it from there (as you did for signIn)